### DISCUSSION TITLE
Usage with lit

### DISCUSSION THREAD

**Discussion by dabukster** [simonwep/viselect#142] [Q&A] [answered]
When is there an example available using lit? 

I'Ve tried to use your lib within a class that extends from LitElement, but it didn't work. I assume this doesn't work due to the shadowDom. 

If there are no plans for a lit example, as a hint, do I have to make the 'document' property work with a document-fragement coming from the shadowRoot, e.g. by "weaken" the deepAssign function?

**Comment by simonwep** [comment, marked as answer]
Support for LitElement is planned, I'll soon push a branch to test it out :)
